MojioSDK/mojiosdksrc/src/main/java/io/moj/mobile/android/sdk/models/Observers/DiagnosticCodeObserver.java

```diff
@@ -1,13 +1,30 @@
-package io.moj.mobile.android.sdk.models.Observers;
+package io.moj.mobile.android.sdk.models.observers;
 
-import io.moj.mobile.android.sdk.models.Diagnostics;
+import com.google.gson.annotations.SerializedName;
 
 import java.util.ArrayList;
 
 /**
+ * Model class for a diagnostic code observer.
  * Created by jian on 01/04/2015.
  */
 public class DiagnosticCodeObserver extends ConditionalObserverBase{
 
-    public ArrayList<Object> DiagnosticCodes;
+    @SerializedName("DiagnosticCodes")
+    private ArrayList<Object> diagnosticCodes;
```

Comment:
Good call, added a TODO. I found some instances of this sort of thing in a few places but the server sometimes expected a String of comma-separated values (meaning we'd need a custom Gson serialization strategy). We'll have to sweep through and verify the server's expectations for each of these so I've mentioned that in the TODO as well